### PR TITLE
Remove Decide and make trackAutomaticEvents required parameter

### PR DESCRIPTION
**Pull request by jaredmixpanel**

### CODE REVIEW

File: Mixpanel-swift.podspec [hunk 1]
```diff
@@ -39,7 +39,7 @@ Pod::Spec.new do |s|
 
   s.subspec 'Complete' do |ss|
     ss.ios.pod_target_xcconfig = {
-      'OTHER_SWIFT_FLAGS' => '$(inherited) -D DECIDE'
+      'OTHER_SWIFT_FLAGS' => '$(inherited) -D AUTOMATIC_EVENTS'
```
Comment (zihejia) [inline Mixpanel-swift.podspec:42]: can we remove this flag and only reply on the setting from `init`?

Comment (jaredmixpanel) [reply Mixpanel-swift.podspec]: replaced with `#if os(iOS) || os(tvOS)`

File: Sources/Track.swift [hunk 2]
```diff
@@ -22,7 +22,10 @@ class Track {
     weak var mixpanelInstance: MixpanelInstance?
     
     var isAutomaticEventEnabled: Bool {
```
Comment (zihejia) [inline Sources/Track.swift:24]: nit: this function can be removed for simplicity.

Comment (jaredmixpanel) [reply Sources/Track.swift]: removed

**Pull request merged**